src/reporter.ts

```diff
@@ -132,11 +132,15 @@ export default class TestRailReporter extends WDIOReporter {
             comment: JSON.stringify(values, null, 1)
         }
 
-        if (this.#options.useCucumber) {
-            if (suiteStats.type === 'scenario') {
-                const testId = suiteStats.title.split(' ')[0].replace('C', '')
-                return this.#api.pushResults(this.runId, testId, results)
+        if (this.#options.useCucumber && suiteStats.type === 'scenario') {
+            let testId = '-1'
+            if (this.#options.caseIdTagPrefix) {
+                // @ts-ignore
```

Comment:
The build is failing because of this, can we make sure we get the types right?

Reply:
@christian-bromann I just saw that and updated it 😄